catalog/spatial/registry/registry-api-impl/src/test/resources/csw-full-registry-package.xml

```diff
@@ -240,7 +240,7 @@
             <rim:ServiceBinding id="urn:registry:federation:method:csw"
                                 service="urn:uuid:service:2014ca7f59ac46f495e32b4a67a51276">
                 <!--Slot defining the query url for this particular method of communicating with this instance-->
-                <rim:Slot name="cswUrl" slotType="xs:anyURI">
+                <rim:Slot name="registryUrl" slotType="xs:anyURI">
```

Comment:
This isn't a registry store but properties for a csw source so its name should stay 'cswUrl'